contact.htm

```diff
@@ -1,46 +1,42 @@
 <!DOCTYPE hmtl>
-<head>
-<meta charset="utf-8" />
-<title>Contact</title>
-<link rel="icon" type="image/x-icon" href="favicon.ico">
-<style>
-
-body {
-	background-image: url("reaper_boarder.jpg");
-	background-repeat: no-repeat;
-	background-attachment: fixed;
-	background-size: cover;
-}
 
-img {
-display: block;
-margin-left: auto;
-margin-right: auto;
-}
-
-</style>
+<head>
+	<meta charset="utf-8" />
+	<title>Contact</title>
+	<link rel="icon" type="image/x-icon" href="favicon.ico">
+	<link rel="stylesheet" href="main.css">
 </head>
 
 <body>
 
-<a href="index.html">
-<img src="metalafterdark_final.jpg" alt="Metal After Dark logo"
-style=width:100%;>
-</a>
-
-<img src="r_contact.png" alt="contact"
-style=width:99%;>
-
-<p style="text-align:center; color:white; font-family:Times New Roman;">
-I appreciate all feedback for Metal After Dark. You are also welcome to suggest albums for me to review, or send reviews of your own.<br> Note that I am unlikely to post your reviews, this is more if you are seeking writing tips or advice. I will also only be reviewing metal albums. This means no nu metal or core genres, hard rock, punk, etc. <br>
-That being said, click the email button below to shoot me an email.<br> I hope you enjoy the site and the reviews it has to offer<br>
--Tobias
-</p>
-<hr>
-
-<a href="https://mail.google.com/mail/u/0/?pli=1#inbox?compose=GTvVlcSGMGJZDDHcmXQVvGDmRjnGkPlntNXLdPZBmDdSDvBRZsPnLbXxtbvFdPWnVWWJdnmjGWSWW">
-<img src="email.jpg" alt="email"
-style="width:50%; border:3px solid #ffffff; padding:2px;">
-</a>
+	<a href="index.html">
+		<img src="images/metalafterdark_final.jpg" alt="Metal After Dark logo" class="logo">
+	</a>
+	<div class="container-width">
+		<img src="images/r_contact.png" alt="contact" class="logo">
+
+		<p class="text-base">
+			I appreciate all feedback for Metal After Dark. You are also welcome to suggest albums for me to review, or send
+			reviews of your own.</p>
+		<p class="text-base">Note that I am unlikely to post your reviews, this is more if you are seeking writing
+			tips
+			or advice. I will also only be reviewing metal albums. This means no nu metal or core genres, hard rock, punk,
+			etc.
+		</p>
+		<p class="text-base">
+			That being said, click the email button below to shoot me an email.</p>
+		<p class="text-base">I hope you enjoy the site and the reviews
+			it
+			has to offer</p>
+		<p class="text-base">
+			-Tobias
+		</p>
+
+		<a
+			href="mailto:setyouremail@here.com">

```

Comment:
The gmail link didn't pre-populate the "To" field for me. The mailto href will ask users which email system they want to user, and will work across devices much better, and will work at all for people who don't use gmail.